### PR TITLE
BP for UAV HUD, enemy assets models, BP for enemy asset spawn

### PR DESCRIPTION
**Pull request by faithfaf**
BP for enemy spawn will need some additional work, but this should keep the camera view and HUD

### CODE REVIEW

File: AirSim/Unreal/Environments/Blocks/Content/Blueprints/BP_Opponent_Random_Asset.uasset
Comment (Cybower) [inline AirSim/Unreal/Environments/Blocks/Content/Blueprints/BP_Opponent_Random_Asset.uasset:1]: What changed on this BP? At a glance, it doesn't seem like anything changed

File: AirSim/Unreal/Environments/Blocks/Content/Maps/FlyingExampleMap.umap
Comment (Cybower) [inline AirSim/Unreal/Environments/Blocks/Content/Maps/FlyingExampleMap.umap:1]: Was there any changes other than placing your custom Blueprints into the level?

File: AirSim/Unreal/Environments/Blocks/Content/Blueprints/BP_Opponent_Random_Asset.uasset
Comment (faithfaf) [reply AirSim/Unreal/Environments/Blocks/Content/Blueprints/BP_Opponent_Random_Asset.uasset:1]: It's for the opponent assets, not ours. It's a copy, but the assets it spawns are the opponent models. I'll then tweak the coordinates

File: AirSim/Unreal/Environments/Blocks/Content/Maps/FlyingExampleMap.umap
Comment (faithfaf) [reply AirSim/Unreal/Environments/Blocks/Content/Maps/FlyingExampleMap.umap:1]: Nope just the blueprints

Comment (Cybower) [reply AirSim/Unreal/Environments/Blocks/Content/Maps/FlyingExampleMap.umap:1]: Ok! What I'll do is pull the Flying Example map back in from main so the changes are merged and then approve the PR 👍🏻

**Review by Cybower — APPROVED** [review]
All good to merge it in 😊

**Pull request merged**